tests/openvino/test_diffusion.py

```diff
@@ -398,6 +408,7 @@ class OVPipelineForImage2ImageTest(unittest.TestCase):
     SUPPORTED_ARCHITECTURES = ["stable-diffusion", "stable-diffusion-xl", "latent-consistency"]
     if is_transformers_version(">=", "4.40.0"):
         SUPPORTED_ARCHITECTURES.append("stable-diffusion-3")
+        SUPPORTED_ARCHITECTURES.append("flux")
```

Comment:
Why is flux addition conditioned on transformers version and not on `is_diffusers_version(">=", "0.31.0")` as above?

Reply:
problem in optimum-intel gone not from diffusers version, in github actions there is division by transformers version, diffusers always used the latest. probmel with models imports comes from saving text encoders that are part of transformers with upper version of tokenizers than supported in transformers < 4.40 as the result test will fail in precommit